lib/cloud/clients/session.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package common
```

Comment:
This is a database specific `Session`. I think we should keep it in `lib/srv/db/*`.

Reply:
I agree. I would keep database access related stuff like session, watchers, etc. where they were.